packages/docs/docs/components/spline.mdx

```diff
@@ -0,0 +1,493 @@
+        [150, 100],
+        [300, 0],
+      ]}
+      end={0}
+    />,
+  );
+
+  yield* spline().end(1, 1.5);
+  yield* spline().start(1, 1.5).to(0.5, 1);
+  yield* spline().end(0.5, 1);
+  yield* all(spline().start(0, 1.5), spline().end(1, 1.5));
+});
+```
+
+### Animating the knots of a spline
+
+Since the `Knot` node is a regular Motion Canvas component, it can be animated
+in much the same way as other components.
+
+:::note
+
+Animating a spline's knots is only possible when using the
+[Knot](#using-knot-nodes) component, not when using the `points` property.
+
+:::
+
+Below are a few examples of interesting effects that can be achieved by
+animating different properties of knots.
+
+<Tabs>
+  <TabItem value='position' label='Position' default>
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  const knotPositions: PossibleVector2[] = [
+    [-200, 0],
+    [-100, -80],
+    [0, 80],
+    [100, -80],
+    [200, 0],
+  ];
+  const knots: Knot[] = [];
+
+  view.add(
+    <Spline lineWidth={6} stroke={'lightseagreen'}>
+      {knotPositions.map((pos, i) => (
+        <Knot ref={makeRef(knots, i)} position={pos} />
+      ))}
+    </Spline>,
+  );
+
+  yield* all(
+    knots[1].position.y(80, 1).to(-80, 1),
+    knots[2].position.y(-80, 1).to(80, 1),
+    knots[3].position.y(80, 1).to(-80, 1),
+  );
+});
+```
+
+  </TabItem>
+
+  <TabItem value="rotation" label="Rotation">
+
+Since `startHandle` and `endHandle` are relative to the knot, rotating the knot
+will also change the handle positions.
+
+:::caution
+
+Note that animating a knot's `rotation` is only possible if its `auto` value is
+less than `1`.
+
+:::
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  const knot = createRef<Knot>();
+
+  view.add(
+    <Spline lineWidth={6} stroke={'lightseagreen'}>
+      <Knot position={[-100, 30]} />
+      <Knot ref={knot} position={[0, -50]} startHandle={[-70, 0]} />
+      <Knot position={[100, 30]} />
+    </Spline>,
+  );
+
+  yield* knot().rotation(360, 3, linear).to(0, 3);
+});
+```
+
+  </TabItem>
+
+  <TabItem value="scale" label="Scale">
+
+Since `startHandle` and `endHandle` are relative to the knot, scaling the knot
+will also change the handle positions.
+
+:::caution
+
+Note that animating a knot's `scale` is only possible if its `auto` value is
+less than `1`.
+
+:::
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  const knot = createRef<Knot>();
+
+  view.add(
+    <Spline lineWidth={6} stroke={'lightseagreen'}>
+      <Knot position={[-100, 30]} />
+      <Knot ref={knot} position={[0, -50]} startHandle={[-70, 0]} />
+      <Knot position={[100, 30]} />
+    </Spline>,
+  );
+
+  yield* knot().scale(3, 2).to(0.2, 2).to(1, 1);
+});
+```
+
+  </TabItem>
+
+</Tabs>
+
+### Animating objects along a spline
+
+Splines are often used to animate objects along a path. The same is possible in
+Motion Canvas by using the `getPointAtPercentage` method of the `Spline`
+component.
```

Comment:
Just a suggestion but I'd reword this as:
```suggestion
Splines can be useful to model the path that an object should follow.
You can use the `getPointAtPercentage` method to achieve this.
```
I think it's a bit clearer since Splines don't do the animating and the current sentence kinda reads like that: "Splines are often used to animate objects [...]"